cg/services/illumina/post_processing/housekeeper_storage.py

```diff
@@ -48,6 +46,12 @@ def _should_fastq_path_be_stored_in_housekeeper(
     )
 
     if metric:
+
+        if sample_is_negative_control_with_reads_in_lane(
```

Comment:
```suggestion
        if is_sample_negative_control_with_reads_in_lane(
```
Nice convention to highlight that it returns a boolean

Reply:
agree